nixos/modules/services/web-apps/nextcloud.nix

```diff
@@ -50,6 +53,13 @@ in {
       default = false;
       description = "Enable if there is a TLS terminating proxy in front of nextcloud.";
     };
+    openFirewall = mkOption {
```

Comment:
IMO it would make sense th have the setting for both modules (with nextcloud passing it to nginx if nginx is enabled). Now if that slows evaluation, that looks bad xD